Update `README.md`. (#211)
yiisoft/db-pgsql: README.md

    <br>
</p>

> Yii DataBase PostgreSQL Extension is a [PostgreSQL] database adapter for the [Yii Framework]. Yii is a high-performance PHP framework for developing web applications. The Yii DataBase PostgreSQL package provides a database connection interface and a set of classes for interacting with a [PostgreSQL] database from within a Yii application. It allows you to perform common database operations such as executing queries, building and executing INSERT, UPDATE, and DELETE statements, and working with transactions. It also provides support for PostgreSQL-specific features such as stored procedures and server-side cursors.

Reviewer: ```suggestion
Yii Database PostgreSQL Extension is a [PostgreSQL] database adapter for the [Yii Framework]. Yii is a high-performance PHP framework for developing web applications. The  package provides a database connection interface and a set of classes for interacting with a [PostgreSQL] database from within a Yii application. It allows you to perform common database operations such as executing queries, building and executing INSERT, UPDATE, and DELETE statements, and working with transactions. It also provides support for PostgreSQL-specific features such as stored procedures and server-side cursors.
```

[![StyleCI](https://github.styleci.io/repos/145220173/shield?branch=master)](https://github.styleci.io/repos/145220173?branch=master)

### Support version

|  PHP | Pgsql Version            |  CI-Actions
|:----:|:------------------------:|:---:|
|**8.0 - 8.2**| **9.0 - 15.0**|[![build](https://github.com/yiisoft/db-pgsql/actions/workflows/build.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-pgsql/actions/workflows/build.yml) [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Fdb-pgsql%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/db-pgsql/master) [![static analysis](https://github.com/yiisoft/db-pgsql/actions/workflows/static.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-pgsql/actions/workflows/static.yml) [![type-coverage](https://shepherd.dev/github/yiisoft/db-pgsql/coverage.svg)](https://shepherd.dev/github/yiisoft/db-pgsql)

### Installation

The package could be installed via composer:

```php
composer require yiisoft/db-pgsql
```

### Config with [Yii Framework]

> The configuration with [container di](https://github.com/yiisoft/di) of [YiiFramework].

Reviewer: ```suggestion
The configuration with [DI container](https://github.com/yiisoft/di) of [Yii Framework].
```

> Also you can use any container di which implements [PSR-11](https://www.php-fig.org/psr/psr-11/).

Reviewer: ```suggestion
Also you can use any DI container which implements [PSR-11](https://www.php-fig.org/psr/psr-11/).
```
